controller/scheduler/swarm.py

```diff
@@ -48,14 +63,20 @@ def stop(self, name):
         """
         Stop a container
         """
-        self.docker_cli.stop(name)
+        try:
+            self.docker_cli.stop(name)
+        except:
+            pass
```

Comment:
It's almost always a bad idea to have a catch-all `except` statement; we should always catch a specific exception type or types.